### PR TITLE
Add simple frontpage, routing and connect detail view with graph

### LINKED ISSUE #31: Create detail view component

### LINKED ISSUE #50: Fix floating nodes
Some nodes are floating around without being linked to others (and they seem to be spaced really far out for some reason).

### LINKED ISSUE #53: Make footer stick to bottom

### LINKED ISSUE #56: navigate between pages
- [ ] Add buttons in navbar
- [ ] connect buttons with routing
- [ ] route images on fronpage to detailview with initialnode

### LINKED ISSUE #57: Implement routing

### LINKED ISSUE #59: Implement frontpage

### PR DESCRIPTION
**Pull request by AndreasWintherMoen**
Closes #59
Closes #58
Closes #57
Closes #56
Closes #53
Closes #50
Closes #31

### CODE REVIEW

File: frontend/src/components/molecules/SustainabilityGoalsList.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  prefix: {
+    prefix: 'SDG',
+    iri: 'http://www.semanticweb.org/aga/ontologies/2017/9/SDG#',
+  },
+  name: 'SDG',
+  id: 'http://www.semanticweb.org/aga/ontologies/2017/9/SDG#SDG',
+};
+
+const SustainabilityGoalsList: React.FC = () => {
+  const [sustainabilityGoals, setSustainabilityGoals] = useState<SustainabilityGoal[]>();
```
Comment (erikskaar) [inline frontend/src/components/molecules/SustainabilityGoalsList.tsx:23]:
```suggestion
  const [sustainabilityGoals, setSustainabilityGoals] = useState<Array<SustainabilityGoal>>();
```
to match the format we've gone with in the rest of the codebase

File: frontend/src/components/pages/About.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const About = () => {
+  const test = { test1: String };
+  console.log(test);
```
Comment (erikskaar) [inline frontend/src/components/pages/About.tsx:5]:
```suggestion
```

File: frontend/src/types/ontologyTypes.ts [hunk 3]
```diff
@@ -5,6 +5,22 @@ export type Annotation = {
...
+
+export type SustainabilityGoals = {
+  sustainabilityGoalList: [
+    {
+      instancesOf: string;
+      label: string;
+      icon: string;
+    },
+  ];
+};
```
Comment (erikskaar) [inline frontend/src/types/ontologyTypes.ts:22]:
```suggestion
export type SustainabilityGoals = Array<SustainabilityGoal>
```

Unless there is a reason for this type to have the list tied to an attribute the object itself should just be an array of SustainabilityGoals

File: frontend/src/components/atoms/OntologyTable.tsx [hunk 4]
```diff
@@ -34,16 +34,13 @@ const renderOntology = (
...
     setSelectedNode(node);
-
-    try {
-      const newOntologies = await getRelations(node.id);
-      setOntologies(newOntologies);
-    } catch (e) {
-      console.log(e);
-    }
+    const newOntologies = await getRelations(node.id);
+    if (newOntologies.length === 0) return;
```
Comment (erikskaar) [inline frontend/src/components/atoms/OntologyTable.tsx:42]: I don't think OntologyTable is necessary anymore and should be yeeted. If it's not going to be yeeted we should dispatch an error here.

**Review by erikskaar — CHANGES_REQUESTED** [review]
Looks good! Some minor changes can be done to improve the code, but overall good work :D

File: frontend/src/components/molecules/SustainabilityGoalsList.tsx (hunk 1 above)
Comment (erikskaar) [reply frontend/src/components/molecules/SustainabilityGoalsList.tsx:23]: Also as we do have a type that is `SustainabilityGoals` then it should be used here as well.

File: frontend/src/types/ontologyTypes.ts (hunk 3 above)
Comment (AndreasWintherMoen) [reply frontend/src/types/ontologyTypes.ts:22]: Is the type necessary at all? Can't we just use Array<SustainabilityGoal> instead?

File: frontend/src/components/molecules/SustainabilityGoalsList.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+import { Link } from 'react-router-dom';
+import { getSustainabilityGoals } from '../../api/ontologies';
+import { mapIdToNode } from '../../common/node';
+import { setError } from '../../state/reducers/apiErrorReducer';
+import { selectNode } from '../../state/reducers/ontologyReducer';
+import { Node, SustainabilityGoal } from '../../types/ontologyTypes';
+import IconContainer from '../atoms/IconContainer';
+
+// Initial node displayed on the frontpage
+const sustainabilityNode: Node = {
```
Comment (AndreasWintherMoen) [inline frontend/src/components/molecules/SustainabilityGoalsList.tsx:13]: This shouldn't be necessary. There's no reason for getSustainabilityGoals to need a node ID as argument.

File: frontend/src/types/ontologyTypes.ts (hunk 3 above)
Comment (erikskaar) [reply frontend/src/types/ontologyTypes.ts:22]: Yeah having a separate type is redundant and just wrapping them in an array where they are used is better practice if I'm not mistaken.

**Review by erikskaar — APPROVED** [review]
LGTM

**Comment by AndreasWintherMoen** [comment]
As a future note, let's try to keep PRs smaller than this one...

**Pull request merged**